easybuild/easyconfigs/m/MCU/MCU-3366c85-gomkl-2021a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'MCU'
```

Comment:
Based on https://hungqpham.com/mcu/index.html, it looks like `mcu` (lowercase) is the most commonly used "spelling" of the software name, so stick to that.